src/main/java/org/web3j/protocol/parity/methods/response/PersonalEcRecover.java

```diff
@@ -0,0 +1,12 @@
+package org.web3j.protocol.parity.methods.response;
+
+import org.web3j.protocol.core.Response;
+
+/**
```

Comment:
The Javadoc comments should provide a sentence describing the purpose of the class. Author details and creation dates are captured in the commit logs.